### PR TITLE
Add Editor Placeholder plugin docs: feature and example

### PR DESCRIPTION
**Pull request by hub33k**
Close https://github.com/ckeditor/ckeditor4/issues/4068

### CODE REVIEW

File: docs/features/editorplaceholder/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+category: inserting-content
```
Comment (Comandeer) [inline docs/features/editorplaceholder/README.md:2]: I'm not sure if it's the correct category. Editor placeholder is not a content, but rather a part of user interface.

File: docs/sdk/examples/editorplaceholder.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+			<p>
+				The <a href="https://ckeditor.com/cke4/addon/editorplaceholder">Editor Placeholder</a> plugin allows you to display a placeholder text when the editor content is empty.
+			</p>
+
+			<textarea cols="10" id="editor1" name="editor1" rows="10" data-sample="1" data-sample-short></textarea>
+
+			<h2>Related Features</h2>
+			<ul>
+				<li><a href="./placeholder.html">Using Placeholders</a></li>
```
Comment (Comandeer) [inline docs/sdk/examples/editorplaceholder.html:45]: Despite the name, this features don't seem related to me: `editorplaceholder` adds a placeholder string to the editable area, while `placeholder` adds uneditable widget containing some dummy content. TBH I can't think of any of our features that are really related to the `editorplaceholder` one :thinking:

File: docs/sdk/examples/editorplaceholder.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+			<p>
+				The <a href="https://ckeditor.com/cke4/addon/editorplaceholder">Editor Placeholder</a> plugin allows you to display a placeholder text when the editor content is empty.
+			</p>
+
+			<textarea cols="10" id="editor1" name="editor1" rows="10" data-sample="1" data-sample-short></textarea>
+
+			<script data-sample="1">
+				CKEDITOR.replace( 'editor1', {
+					extraPlugins: 'editorplaceholder',
```
Comment (hub33k) [inline docs/sdk/examples/editorplaceholder.html:45]: Is this necessary here? I think we could remove this line :thinking:

Comment (Comandeer) [reply docs/sdk/examples/editorplaceholder.html:45]: If it works without this line, then yes. I don't exactly remember which preset is used in docs, but `editorplaceholder` is included only in `full` one, so if we use it in docs, it should work.

Comment (hub33k) [reply docs/sdk/examples/editorplaceholder.html:45]: Ok, nevermind. We need this line here in order to this example work.

**Comment by Comandeer** [comment]
As it's a new feature, I'd like to ask @godai78 for a final review.

File: docs/features/editorplaceholder/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+---
+<!--
+Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Using Editor Placeholder
+
+<info-box info="">
+	This feature was introduced in CKEditor 4.15 and is provided through the <a href="https://ckeditor.com/cke4/addon/editorplaceholder">Editor Placeholder</a> plugin that is included in the Standard and Full presets available from the official CKEditor <a href="https://ckeditor.com/ckeditor-4/download/">Download</a> site. You can also {@link guide/dev/plugins/README add it to your custom build} with <a href="https://ckeditor.com/cke4/builder">online builder</a>.
```
Comment (godai78) [inline docs/features/editorplaceholder/README.md:16]: Are capitalized names still a thing? I thought we were not using it unless this is a specifically named feature.

Comment (godai78) [inline docs/features/editorplaceholder/README.md:16]:
```suggestion
	This feature was introduced in CKEditor 4.15 and is provided through the <a href="https://ckeditor.com/cke4/addon/editorplaceholder">Editor Placeholder</a> plugin that is included in the Standard and Full presets available from the official CKEditor <a href="https://ckeditor.com/ckeditor-4/download/">download</a> site. You can also {@link guide/dev/plugins/README add it to your custom build} with the <a href="https://ckeditor.com/cke4/builder">online builder</a>.
```

Comment (hub33k) [reply docs/features/editorplaceholder/README.md:16]: Yes, we should be consistent with rest of docs.

Comment (Comandeer) [reply docs/features/editorplaceholder/README.md:16]: It seems that rest of docs still use capitalized names for both plugins and Download site name :thinking:. So to be consistent we should also keep it here.

**Pull request merged**